tempesta_fw/http.c

```diff
@@ -2369,7 +2373,7 @@ tfw_http_req_process(TfwConn *conn, struct sk_buff *skb, unsigned int off)
 		 * from this point on.
 		 */
 		if (req_conn_close)
-			return TFW_STOP;
+			return TFW_BLOCK;
```

Comment:
I think it's better to replace this by

     if (req->flags & TFW_HTTP_CONN_CLOSE)
         break;

since `TFW_BLOCK ` means blocked (typically malicious) request and it's better to keep value from the previous call .

Reply:
Corrected partially (reason described in previous comment).